libs/application/data-providers/src/providers/PaymentCatalogProvider.ts

```diff
@@ -45,39 +45,34 @@ export class PaymentCatalogProvider extends BasicDataProvider {
 
     return this.useGraphqlGateway(query, {
       input: { performingOrganizationID: '6509142520' },
```

Comment:
just to note, that probably need to be move as env variable and/or secret

Reply:
well, this will be application dependant eventually, yes, definitely a very temporary thing